apiserver/cmd/main.go

```diff
@@ -165,8 +165,27 @@ func startHttpProxy() {
 		topMux = http.NewServeMux()
 	}
 
-	// Seems /apis (matches /apis/v1alpha1/clusters) works fine
-	topMux.Handle("/", runtimeMux)
+	if allowedOrigin := os.Getenv("CORS_ALLOW_ORIGIN"); allowedOrigin != "" {
```

Comment:
This seems to be a breaking change for users when users use their old `Values.yaml` with the new api server image.

The same YAML with old image will enable CORS, but CORS will be disabled with the new image.

Reply:
What do you mean? If users use old `values.yaml`, then `CORS_ALLOW_ORIGIN` won't be set, and the `else` block will be executed. So `topMux.Handle("/", runtimeMux)` will still be run. There is no behavior change.